bson/src/test/unit/org/bson/codecs/pojo/Student.java

```diff
@@ -0,0 +1,6 @@
+package org.bson.codecs.pojo;
```

Comment:
Should this be moved to the same package as all the other records?

Reply:
This actually isn't used by any of the tests. I was using it to manually test something else and forgot to delete it. Deleted now.